### DISCUSSION TITLE
Mi band 7 support

### DISCUSSION THREAD

**Discussion by LOSEARDES77** [NightscoutFoundation/xDrip#2795] [Q&A] [answered]
Hi,
 i just wanted to know if mi band 7 will be supported because at the moment it seams to not work.
i got a new mi band 7 a few days ago and i tried to do the tutorial steps as it was with the mi band 6 but it didn't work.
Thanks!

**Comment by AliQ101** [comment, marked as answer]
Mi band 7 is not supported by watchdrip+ app as fetch API feature was not implemented by xiaomi in this watch type. Amazfit band 7 is supported.

**Reply by LOSEARDES77** [reply]
ok Thanks!
